docs/cvl/statements.md

```diff
@@ -127,10 +131,14 @@ some arbitrary state.  It is not possible to check that every possible starting
 state has an execution that satisfies the condition.
 ```
 
+- [`satisfy` example](https://github.com/Certora/Examples/blob/14668d39a6ddc67af349bc5b82f73db73349ef18/CVLByExample/ConstantProductPool/certora/spec/ConstantProductPool.spec#L243)
+
 (requireInvariant)=
 `requireInvariant` statements
 -----------------------------
 
+- [`requireInvariant` example](https://github.com/Certora/Examples/blob/14668d39a6ddc67af349bc5b82f73db73349ef18/CVLByExample/ConstantProductPool/certora/spec/ConstantProductPool.spec#L178)
+
```

Comment:
remove the todo here , the example is enough. maybe add a line that says that it is short for require and the expression of the invariant where parameters have to be bind

Reply:
done